src/gui/dialogues/newflightdialog.cpp

```diff
@@ -255,6 +295,8 @@ void NewFlightDialog::setupRawInputValidation()
             }
         }
     }
+
+    // [G]: TODO cleanup
     // populate Mandatory Line Edits list and prepare QBitArray
```

Comment:
Is something working on your side already? At the moment the dialog is inop for me, crashes with a 
`ASSERT: "uint(i) < uint(size())" in file /usr/include/x86_64-linux-gnu/qt5/QtCore/qbitarray.h, line 125`,
probably due to the bit array not being initialized?

Reply:
Fixed
Apparently QVector has `resize` just like QBitArray. 
```c++
lineEdits = init_list;
lineEdits.resize(init_list.size());
```
forgot the lineEdits**Valid**